src/content/reference/cli.mdx

```diff
@@ -709,8 +709,9 @@ The `status` command should be run from the same location than `okteto up`.
 
 ### up
 
-Activate a development container.
-If needed, `okteto up` builds the images and runs the deploy commands defined in your [Okteto Manifest](reference/manifest.mdx).
+The `okteto up` command is one of Okteto’s key features. This command uses the [Okteto Manifest](/reference/manifest) you define to build, deploy, and synchronize your local files with the remote development container.
+
+This command has two modes: `sync` (default mode) and `hybrid` which are detailed in the manifest reference: [mode](/reference/manifest#mode-string-optional).
```

Comment:
Is the link to `mode` broken? In the preview, I cannot be redirected to that section

Reply:
```suggestion
This command has two modes: `sync` (default mode) and `hybrid` which are detailed in the manifest reference: [mode](/reference/manifest/#mode-string-optional).
```
It was broken due to a missing `/` 🤦‍♂️ 

Great catch!